2-WebApp-graph-user/2-5-HybridFlow/wwwroot/js/site.js

```diff
@@ -0,0 +1,9 @@
+﻿async function signOut() {
+    msalInstance.logoutRedirect();
+
+    fetch('/logout', {
+        method: 'POST',
+        mode: 'same-origin',
+        credentials: 'same-origin',
+    });
```

Comment:
This will perform a redirect, and as such there is a not a guarantee the fetch below will get run. I would swap the order and await the fetch call to ensure both run.

```suggestion
    await fetch('/logout', {
        method: 'POST',
        mode: 'same-origin',
        credentials: 'same-origin',
    });
    
    await msalInstance.logoutRedirect();
```